cpp/ScanChain_Zp.C

```diff
@@ -569,20 +722,24 @@ int ScanChain(TChain *ch, double genEventSumw, TString year, TString process) {
       for ( unsigned int iplot=0; iplot < plot_names.size(); iplot++ )
```

Comment:
I would probably move this histo filling together with the one on L865, so that we can have a full picture of all of the histos filled for sel5.